pytorch_lightning/core/datamodule.py

```diff
@@ -134,13 +136,20 @@ def test_dataloader():
                 return [dataloader(ds) for ds in test_dataset]
             return dataloader(test_dataset)
 
+        def predict_dataloader():
```

Comment:
Should we have a test case for this change as we have ones for `{train,val,test}`?

https://github.com/PyTorchLightning/pytorch-lightning/blob/9bfbd9ea80c4c5daf48800a1ab9a14683a0bc1ce/tests/core/test_datamodules.py#L400-L424

Reply:
good catch!

Reply:
done!